src/client/components/footer/footer.style.css

```diff
@@ -1,5 +1,13 @@
+/* TODO: to be deleted later when we use use global style */
+* {
+  margin: 0;
+  padding: 0;
+  color: #3b4349;
+  font-family: Proxima Nova, sans-serif;
+}
+
 .footer {
-  /* TODO: background-color to be deleted later when we use global style */
+  /* TODO: background-color to be deleted later when we use use global style */
   background-color: #ebecf0;
```

Comment:
Haha... I think it was a typo but you added another "use". Can't you put the background color of the footer in the theme file now? And remove the TODO, obviously.

Reply:
Not mine code, but yes...